### DISCUSSION TITLE
How to turn off FPS Cap?

### DISCUSSION THREAD

**Discussion by Jimbo33333** [sultim-t/xash-rt#133] [General]
My game is capped at like 63 fps is there a way to disable that?

**Comment by Jimbo33333** [comment]
Nevermind found fix

**Reply by CubaOfficial** [reply]
How?

**Reply by LeGrosWinnie** [reply]
There is no fix... On my 3070 Ti, 12600K, 32 GB RAM, the game/mod runs at 45 fps...
It's not a "fps cap" it's just the game running low fps... Evne on 4XXX series it wont go above 60 fps or so.
